docs/docs/references/users/README.md

```diff
@@ -261,3 +261,54 @@ Every time the user signs in with a social connector,
 their _identities_ will be automatically imported or updated from the identity provider.
 
 :::
+
+## Session and cookie
+
+A session is a group of interactions between a user and an application that takes place (maybe on different pages) within a period of time.
+A single session temporarily stores multiple interactions (such as sign-in, linking to a social network, consent authorization, and sign-out) before it ends.
+If the user leaves a website or closes the browser, the session will end by default.
```

Comment:
i don;t think we need this.. our reader has the basic concept of session or cookie or they can just go wikipedia. we just need to talk about how session and cookie works in logto.

Reply:
Can I add a link for the freshman to read?

Reply:
reorg